packages/TorneloScoresheet/src/pages/GraphicalRecording/GraphicalRecording.tsx

```diff
@@ -32,14 +36,20 @@ const GraphicalRecording: React.FC = () => {
     goToArbiterMode();
   };
 
+  const position = graphicalRecordingMode?.board
```

Comment:
dont you think it makes more sense to have this logic inside the chessboard component itself and just have the chess board component take a `ChessBoardPosition` object instead?